src/transferslot.cpp

```diff
@@ -370,10 +389,36 @@ void TransferSlot::doio(MegaClient* client)
     {
         if (reqs[i])
         {
+            unsigned slowestConnection;
+            if (transfer->type == GET && reqs[i]->contentlength == reqs[i]->size)
+                if (transferbuf.detectSlowestRaidConnection(i, slowestConnection))
```

Comment:
I think this `if` should be replaced by one `&&` in the previous `if`. It makes it more readable and less prone to error if touched in the future.

Reply:
agreed, and done.  I had arranged it like that temporarily so I could breakpoint the last condition, and had not put it back :-)  cheers